roles/jbcs/tasks/install.yml

```diff
@@ -72,7 +72,6 @@
   delegate_to: localhost
   run_once: true
   when:
-    - not jbcs_installed is defined
```

Comment:
variable needs to be documented in README and argument_specs (btw, where is it defined?)

Reply:
It's from the main. And it is not documented nor needed, so that's why I removed it.

Reply:
a-ha I see, no where where it came from